inf-clojure.el

```diff
@@ -208,6 +209,21 @@ whichever process buffer you want to use.")
 
 (put 'inf-clojure-mode 'mode-class 'special)
 
+(defcustom inf-clojure-repl-flavor 'clojure
```

Comment:
So I am beginning to understand a bit more, a `defcustom` is global and therefore this will be set once and forever right? So this becomes buffer-local? I need to read more on this...and thanks for reviewing!